packages/utils/package.json

```diff
@@ -29,12 +29,12 @@
     "@pixi/constants": "6.0.0",
     "@pixi/settings": "6.0.0",
     "@types/earcut": "^2.1.0",
-    "@types/node": "^12.0.0",
     "earcut": "^2.2.2",
     "eventemitter3": "^3.1.0",
     "url": "^0.11.0"
   },
   "devDependencies": {
+    "@types/node": "^12.0.0",
```

Comment:
Do we even need this anymore?

Reply:
They provide some sanity checking for the assignment of `_xxx` functions to the redeclared `xxx` ones  in the url wrapper file. You can try for example to change the `ResolveFunction` return type to `number` and the compiler will complain about `parse = _parse` as the types are incompatible. Removing the Node types completely would also remove this type check.

Reply:
I guess it's a choice between having those wrappers type checked and having the Node types leaking all over the `utils` package or removing type checking from the wrappers and fixing the Node types leak. It could actually be that removing the Node types would have more pros - It's not like those types will be changed often.

Reply:
Yes exactly. We are just using a very narrow scope of those APIs. I'm for removing node types for less complexity and just casting url internally for more strict types.

Reply:
Agreed. I pushed a commit that removes the Node types completely.